src/Groups/homomorphisms.jl

```diff
@@ -346,17 +346,17 @@ function image(f::GAPGroupHomomorphism)
 end
 
 """
-    image(f::GAPGroupHomomorphism{S, T}, H::S) where S <: GAPGroup where T <: GAPGroup
-    (f::GAPGroupHomomorphism{S, T})(H::S)
+    image(f::GAPGroupHomomorphism{S, T}, H::GAPGroup) where {S <: GAPGroup, T <: GAPGroup}
+    (f::GAPGroupHomomorphism{S, T})(H::GAPGroup)
```

Comment:
We don't say what `S`, `T` are here...

We can also write this without `where`:
```suggestion
    (f::GAPGroupHomomorphism{<:GAPGroup, <:GAPGroup})(H::GAPGroup)
```